socrates_api/serializers.py

```diff
@@ -617,15 +617,40 @@ def delete(self):
     def create_link(self, instance):
         return reverse('socrates_api:firewall_addressgroup_detail', kwargs={'slug': instance['name']}, request=self.context.get('request'))
 
+def validate_port_number(port):
+    if 0 < port >= 65535:
```

Comment:
This should be 0 < port <= 65535, right?

Reply:
oh shoot! to eager to press shift :)